src/utils/Api.ts

```diff
@@ -17,7 +17,7 @@ export default class Api {
   private chainId: string
 
   constructor(abis: HexAbi[], publicKeys: string[], chainId: string) {
-    this.rpc = new JsonRpc(null) // provide null chainURL to ensure vault isn't reaching out to chain for info
+    this.rpc = new JsonRpc('') // provide empty chainURL to ensure vault isn't reaching out to chain for info
```

Comment:
Can we just remove this value altogether?  This still makes me uncomfortable because it looks like you *have* to pass an empty string where we used to have to pass null.  If there's no value, there should just be no value, and we should default the value or deal with no value internally.  Using a value as a non-value is not high on the coolness scale... :)